Kickstarter-iOS/Views/Controllers/DiscoveryPageViewController.swift

```diff
@@ -98,6 +95,11 @@ internal final class DiscoveryPageViewController: UITableViewController {
 
     _ = self
       |> baseTableControllerStyle(estimatedRowHeight: 200.0)
+
+    if let preferredBackgroundColor = self.preferredBackgroundColor {
```

Comment:
`baseTableControllerStyle` always sets the background to `white`, this was necessary for us to control it from the parent.